feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailScreen.kt

```diff
@@ -72,9 +94,13 @@ fun MovieDetailScreen(
     MovieDetailContent(
       videoDetail = videoDetail,
       state = state,
+      creditState = creditState,
       onBackPressed = onBackPressed,
       onPlayPressed = viewModel::loadStreamInfo,
       onAddToHistoryPressed = viewModel::addItemToHistory,
+      similarState = similarState,
+      rateColor = viewModel.getRateColor(),
```

Comment:
No need to pass rateColor here, if you use the extension function I mentioned earlier